src/components/Page/components/Header/components/PageTitle/PageTitle.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../../../../../styles/common';
+
+.hasThumbnail {
+  display: grid;
+  grid-gap: spacing();
+  grid-template-columns: auto 1fr;
+}
+
+.Title {
+  @include text-breakword;
+}
+
+.Title-hasMetadata {
+  margin-right: spacing(tight);
+  display: inline;
+
+  & > * {
```

Comment:
Since we want the metadata to be inline and DisplayText is block by default I needed to do this :(

Another option would be to simply make this an `H1` and style it myself